app/controllers/dor_controller.rb

```diff
@@ -34,14 +36,24 @@ def delete_from_index
 
   private
 
+  def cocina_hash
+    @cocina_hash ||=
+      if params[:cocina_object].present?
+        # params[:cocina_object] is itself an ActionController::Parameters instance, #permit! is needed to allow the #to_h
+        params[:cocina_object].permit!.to_h
+      else
+        {}
```

Comment:
This is going to result in an error when you try to pass it to the builder, so why not just skip this conditional and do `params.require(:cocina_object)`?

Reply:
ah yeah, good point